cws-core/src/main/java/jpl/cws/core/db/DbService.java

```diff
@@ -9,6 +9,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+//import javax.annotation.Resource;
```

Comment:
me just being picky, but no need to keep lines like this, in the code

Reply:
Removed